db/migrate/20160426174031_create_companies.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCompanies < ActiveRecord::Migration
+  def change
+    create_table :companies do |t|
+      t.string :name
```

Comment:
I think this can be a part of Give admin ability to edit company data, what do you think?